mmv1/third_party/terraform/tests/resource_spanner_database_test.go.erb

```diff
@@ -117,6 +117,112 @@ resource "google_spanner_database" "basic" {
 `, instanceName, instanceName, databaseName)
 }
 
+func TestAccSpannerPGDatabase_basic(t *testing.T) {
+	t.Parallel()
+
+	project := getTestProjectFromEnv()
+	rnd := randString(t, 10)
+	instanceName := fmt.Sprintf("tf-test-%s", rnd)
+	databaseName := fmt.Sprintf("tfgen_%s", rnd)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckSpannerDatabaseDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSpannerPGDatabase_basic(instanceName, databaseName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("google_spanner_database.basic_spangres", "state"),
+				),
+			},
+			{
+				// Test import with default Terraform ID
+				ResourceName:            "google_spanner_database.basic_spangres",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"ddl", "deletion_protection"},
+			},
+			{
+				Config: testAccSpannerPGDatabase_basicUpdate(instanceName, databaseName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("google_spanner_database.basic_spangres", "state"),
+				),
+			},
+			{
+				// Test import with default Terraform ID
+				ResourceName:            "google_spanner_database.basic_spangres",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"ddl", "deletion_protection"},
+			},
+			{
+				ResourceName:            "google_spanner_database.basic_spangres",
+				ImportStateId:           fmt.Sprintf("projects/%s/instances/%s/databases/%s_spangres", project, instanceName, databaseName),
```

Comment:
For this test case, you can remove the import steps w/ `ImportStateId` in them. Being postgres-based doesn't affect the id, so we're not getting additional information out of the extra steps.